flink-connectors/flink-connector-kafka-0.8/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer081.java

```diff
@@ -27,6 +28,7 @@
  * @deprecated Use {@link FlinkKafkaConsumer08}
  */
 @Deprecated
+@PublicEvolving
```

Comment:
I don't think this annotation is needed here, since the class is deprecated already.